Makefile

```diff
@@ -36,6 +38,21 @@ setup:
 
 .PHONY: build
 build: makes
+	go vet cmd/main_postsubmit.go
+	go run cmd/main_postsubmit.go \
+		--target=build \
+		--release-branch=${RELEASE_BRANCH} \
+		--release=${RELEASE} \
+		--development=${DEVELOPMENT} \
+		--region=${AWS_REGION} \
+		--account-id=${AWS_ACCOUNT_ID} \
+		--base-image=${BASE_IMAGE} \
+		--image-repo=${IMAGE_REPO} \
+		--go-runner-image=${GO_RUNNER_IMAGE} \
+		--kube-proxy-base=${KUBE_PROXY_BASE_IMAGE} \
+		--artifact-bucket=$(ARTIFACT_BUCKET) \
+		--upload-to-s3=true \
```

Comment:
Will this upload step not fail in the `s3_sync` script, since we do not have a bucket of that name in the account?

Reply:
the prow job should be passing in the postsubmit bucket name. I don't recall what bucket that is.